openedx/features/edly/tests/test_utils.py

```diff
@@ -0,0 +1,36 @@
+import jwt
+from django.conf import settings
+from django.test import TestCase
+
+from openedx.features.edly.utils import encode_edly_user_info_cookie, decode_edly_user_info_cookie
+
+class UtilsTests(TestCase):
+    """
+    Tests for utility methods
+    """
+
+    def test_encode_edly_user_info_cookie(self):
+        """
+        Test that encode_edly_user_info_cookie method encodes data correctly.
+        """
+        test_data = {
+            'some_data': 'test data'
+        }
+        actual_encoded_string = encode_edly_user_info_cookie(test_data)
+        expected_encoded_string = jwt.encode(test_data, settings.EDLY_COOKIE_SECRET_KEY, algorithm=settings.EDLY_JWT_ALGORITHM)
+
+        assert actual_encoded_string == expected_encoded_string
+
+    def test_decode_edly_user_info_cookie(self):
+        """
+        Test that decode_edly_user_info_cookie method decodes data correctly.
+        """
+        test_data = {
+            'some_data': 'test data'
+        }
+        encoded_data = jwt.encode(test_data, settings.EDLY_COOKIE_SECRET_KEY, algorithm=settings.EDLY_JWT_ALGORITHM)
+
+        actual_decoded_string = decode_edly_user_info_cookie(encoded_data)
+        expected_decoded_string = jwt.decode(encoded_data, settings.EDLY_COOKIE_SECRET_KEY, algorithms=[settings.EDLY_JWT_ALGORITHM])
```

Comment:
I believe the decoding of encoded dict should be a dict.

`encode_edly_user_info_cookie` method encode dict data in a hash
`decode_edly_user_info_cookie` method decodes hash back to dict

Reply:
The `jwt.decode()` method automatically returns `dict` if the encoded string was of a `dict`.

Reply:
We don't need such complicated testing. I have added my feedback